src/reports/tables/Eight.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const renderData = applicantCharacteristics => {
+  return applicantCharacteristics.map((characteristic, index) => {
+    return [
+      <tr key={index}>
+        <th
+          style={{
+            borderTopWidth: '2px',
+            fontWeight: 'bold',
+            textTransform: 'uppercase',
+            backgroundColor: '#f1f1f1'
+          }}
+          colSpan={21}
+        >
+          {characteristic.characteristic}
+        </th>
+      </tr>,
+      renderDetails(characteristic, index)
+    ]
+  })
+}
+
+const renderDetails = (characteristic, key) => {
+  let characteristicDetails = ''
+  if (characteristic.characteristic === 'race') characteristicDetails = 'races'
+  if (characteristic.characteristic === 'ethnicity')
+    characteristicDetails = 'ethnicities'
+  if (characteristic.characteristic === 'minorityStatus')
+    characteristicDetails = 'minorityStatuses'
+  if (characteristic.characteristic === 'gender')
+    characteristicDetails = 'genders'
+  if (characteristic.characteristic === 'income')
+    characteristicDetails = 'incomes'
+
+  return characteristic[characteristicDetails].map((detail, index) => {
+    return (
+      <tr key={`${index}-${key}`}>
+        <th>{detail[characteristic.characteristic]}</th>
+        {renderDenialReasons(detail.denialReasons, index, key)}
+      </tr>
+    )
+  })
+}
+
+const renderDenialReasons = (denialReasons, key, key2) => {
+  return denialReasons.map((denialReason, index) => {
+    return [
+      <td key={`count-${index}-${key}-${key2}`}>{denialReason.count}</td>,
+      <td key={`percentage-${index}-${key}-${key2}`}>
+        {denialReason.percentage}
+      </td>
+    ]
+  })
+}
+
+const Eight = props => {
+  if (!props.report) return null
+
+  return (
+    <table style={{ fontSize: '.75em' }}>
```

Comment:
This table is large enough that it will need a style to reduce its padding. A rule like the following fixes the horizontal overflow:
```
.largeTable td, .largeTable th {
  padding: 1rem;
}
```